### DISCUSSION TITLE
part-ii/2020-24/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#164] [Question comments]
Part II 2020 Automata and Formal Languages: Paper 3, Section I, F 

https://questions.tripos.org/part-ii/2020-24/

**Comment by artie2000** [comment]
<details>
<summary>(i) Context-free:</summary>

$S\to a^2Sb^2\mid\varepsilon$

</details>

(ii) No: use the pumping lemma.
